### PR TITLE
revisions

### PR DESCRIPTION
**Pull request by lgozasht**

### CODE REVIEW

File: scripts/Fasta2UShER.py [hunk 1]
```diff
@@ -65,46 +93,67 @@
...
+            args['user_specified_mask'], head, msaName, args['output'], filePath))
+
+
+# Else run fastaToVcf without masking
 
 else:
 
     if '.vcf' in args['output']:
-        os.system('{3}/faToVcf  -ref=\"{0}\" {1} {2}'.format(head,msaName,args['output'],filePath))
+        os.system('{3}/faToVcf  -ref=\"{0}\" {1} {2}'.format(head,
```
Comment (yatisht) [inline scripts/Fasta2UShER.py:148]: I think Art also wanted that we get rid of os.system calls

Comment (lgozasht) [reply scripts/Fasta2UShER.py:148]: I thought it was just for rm -r on temp files

Comment (lgozasht) [reply scripts/Fasta2UShER.py:148]: will replace with subprocess tho

**Comment by lgozasht** [comment]
all os.system commands replaced with subprocess.call

**Comment by yatisht** [comment]
This is all good but just to make it air-tight, would you mind adding a few more comments to the inner if-else statements (like add '.vcf' extension if not specified etc.)?

**Comment by lgozasht** [comment]
Hows this?

**Comment by yatisht** [comment]
You may want to ensure PEP8 compliance again

**Comment by lgozasht** [comment]
ugh yeah whoops

**Comment by yatisht** [comment]
Also fixed typos in comments (extention -> extension) and fastaToVcf -> faToVcf

**Comment by yatisht** [comment]
You might want to use this style for printing long strings: https://www.techbeamers.com/python-multiline-string/

**Comment by lgozasht** [comment]
this just applies to that first print statement right?

**Comment by yatisht** [comment]
That and maybe the parser description too.

**Comment by lgozasht** [comment]
okay

**Pull request merged**